langworld_db_data/adders/listed_value_adder.py

```diff
@@ -89,6 +131,57 @@ def _add_to_inventory_of_listed_values(
 
         return id_of_new_value
 
+    @staticmethod
+    def _get_indices_and_their_line_numbers_in_features_listed_values(
+        rows: list[dict[str, str]],
+        feature_id: str,
+        new_value_en: str,
+        new_value_ru: str,
```

Comment:
variable names are misleading: the function is supposed to just get indices, and suddenly you're asked to pass in some "new value"

Reply:
These are necessary for validation:
```python
if row["en"] == new_value_en or row["ru"] == new_value_ru:
    raise ValueError(f"Row {row} already contains value you are trying to add")
```
Should it be moved away from this method or should I just rename the arguments?